src/leap/common/events/client.py

```diff
@@ -55,275 +54,269 @@ class CallbackAlreadyRegistered(Exception):
     pass
 
 
-def ensure_client_daemon():
+class EventsIOLoop(ioloop.ZMQIOLoop):
```

Comment:
does this mean that all clients will use the zmqioloop?
isn't this only needed for the qt frontend?